### PR TITLE
Add libexif

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/libexif/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+  run:
```
Comment (marcelotrevisani) [inline recipes/libexif/meta.yaml]: Could you please remove those empty sections?

File: recipes/libexif/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+test:
+  commands:
```
Comment (marcelotrevisani) [inline recipes/libexif/meta.yaml]: Could you please add some tests here?
Is it generating some shared libraries?
If so you can test it like:
```suggestion
  commands:
    - test -f ${PREFIX}/lib/libexif${SHLIB_EXT}   # [unix]
```

File: recipes/libexif/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ 
+about:
+  home: https://libexif.github.io/
+  license: LGPL-2.1
+  license_file: COPYING
+  summary: 'A library for parsing, editing, and saving EXIF data'
```
Comment (marcelotrevisani) [inline recipes/libexif/meta.yaml]: Could you please add the ``license_file`` ?

File: recipes/libexif/meta.yaml (hunk 1 above)
Comment (sebastian-luna-valero) [reply recipes/libexif/meta.yaml]: Sure.

File: recipes/libexif/meta.yaml (hunk 2 above)
Comment (sebastian-luna-valero) [reply recipes/libexif/meta.yaml]: Great addition, many thanks!

**Pull request merged**